library-management-system/pom.xml

```diff
@@ -2,22 +2,28 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
-    <groupId>com.johnny</groupId>
-    <artifactId>projects-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
-    <packaging>pom</packaging>
+    <parent>
+        <groupId>com.johnny</groupId>
+        <artifactId>projects-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
 
-    <properties>
-        <maven.compiler.source>18</maven.compiler.source>
-        <maven.compiler.target>18</maven.compiler.target>
-    </properties>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>library-management-system</artifactId>
+    <packaging>jar</packaging>
 
-    <modules>
-        <module>abstract-class</module>
-        <module>interval</module>
-    </modules>
+    <dependencies>
+        <dependency>
```

Comment:
Are you using the abstract-class module in the library management system? If not, can you please remove it from the pom file? Same goes for the guava dependency.
FYI (for your info), it is better to have a clean pom file to avoid increasing the build time, as well as the runtime of the application. This also helps in maintaining your code for the future. There are some plugins that fail the build in case you added a dependency that is not used (but that's not the purpose of this exercise).